src/pint/models/stand_alone_psr_binaries/ELL1_model.py

```diff
@@ -46,10 +46,10 @@ def ttasc(self):
         """
         ttasc = t - TASC
         """
+        t = self.t
         if not hasattr(self.t, "unit") or self.t.unit == None:
             t = self.t * u.day
-        t = self.t
-        ttasc = (t - self.TASC).to("second")
+        ttasc = (t - self.TASC.value * u.day).to("second")
```

Comment:
Isn't `self.TASC.quantity` a Quantity? If that has units of days then you could drop `* u.day`, and if it doesn't then `.value*u.day` will give the wrong answer?

Reply:
That doesn't work because you can't subtract Time quantities like that form non Time quantities.  Here is an example:
```
In [7]: 50000.0*u.day - mm.TASC.quantity                                                                 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-01b65069e5a2> in <module>
----> 1 50000.0*u.day - mm.TASC.quantity

~/python_venvs/py3/lib/python3.8/site-packages/astropy/time/core.py in __rsub__(self, other)
   2016     def __rsub__(self, other):
   2017         out = self.__sub__(other)
-> 2018         return -out
   2019 
   2020     def _time_comparison(self, other, op):

TypeError: bad operand type for unary -: 'Time'
```

Reply:
And self.t is a numpy array of non-Time quantities